File: app/src/main/java/com/rivera/dispositivostest/data/entities/repository/DBUsers.java
package com.rivera.dispositivostest.data.entities.repository;

public class DBUsers {

    fun getListUsers(){
        var usr1 = Users(1,"admin","admin")
        var usr2 = Users(2,"juan","juan")
        var usr3 = Users(3,"maria","maria")

        var listUsers = listOf(usr1, usr2,usr3)
        return listUsers
    }
}

File: app/src/main/java/com/rivera/dispositivostest/data/entities/repository/DBUsers.kt
package com.rivera.dispositivostest.data.entities.repository

import com.rivera.dispositivostest.data.entities.Users

    class DBUsers {
        fun getListUsers(): List<Users> {


            return listOf<Users>(
                Users("admin", "admin", 1),
                Users("juan","juan", 2),
                Users("maria","maria", 3),
                Users("Cristina","12345", 4),

            )


        }
    }
